mlflow/server/js/package.json

```diff
@@ -24,7 +24,9 @@
     "extract-i18n:check": "echo '\n*******************\n`yarn extract-i18n:check` command is now deprecated. Please use `yarn i18n:check` instead.'; exit 1",
     "extract-i18n": "echo '\n*******************\n`yarn extract-i18n` command is now deprecated. Please use `yarn i18n` instead.'; exit 1",
     "check-all": "yarn gocx-codegen:check && yarn type-check && yarn lint && yarn i18n:check",
-    "fix-all": "yarn gocx-codegen && yarn i18n && yarn lint:fix && yarn type-check",
+    "fix-all": "yarn gocx-codegen && yarn graphql-codegen && yarn i18n && yarn lint:fix && yarn type-check",
+    "graphql-codegen": "yarn graphql-codegen:clean && graphql-codegen --config ./src/graphql/graphql-codegen.ts && eslint src/graphql/__generated__/*.ts --fix",
```

Comment:
Hmm do these actually work? We don't have any graphql in oss yet right? maybe we should wait to introduce?

Reply:
Yeah, I'm ok with removing those for now